sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -609,6 +609,22 @@ def test_pred_int_tag(self, estimator_instance):
         if isinstance(f, _DelegatedForecaster):
             return None
 
+        # we skip the _DelegatedForecaster, since it implements delegation methods
+        #   which may look like the method is implemented, but in fact it is not
+        if isinstance(f, _DelegatedForecaster):
```

Comment:
Why are there these changes?

Reply:
It migth come from a mistake when merging branches. I will delete it.